src/tools/dart-run.md

```diff
@@ -0,0 +1,95 @@
+---
+title: dart run
+deprogramion: Use dart run to run a Dart program in your package.
+---
+
+The `dart run` command supports running a Dart program located in a file,
+or a package, or in one of its dependencies, from the command line.
+
+To run an executable when you are not currently inside a package,
+use the [pub global](/tools/pub/cmd/pub-global) command.
+
+The `dart run` command combines what was previously exposed as `pub run`
+and as the [Dart VM](/tools/dart-vm).
+
+{% include tools/dart-tool-note.md %}
+
+Here's an example of creating a new app and running it:
+
+```terminal
+$ dart create myapp
+$ cd myapp
+$ dart run
+```
+
+## Running a program in a Dart file
+
+This is the simplest form, where you pass a relative path to a `.dart` file.
+
+To run a program in the file `internal/debug.dart`,
```

Comment:
nit: `test/debug.dart` might be better... `internal/` is not a convention.

Reply:
done